src/app/pcasts/models/user.py

```diff
@@ -7,6 +7,7 @@ class User(Base):
 
```

Comment:
We should add a custom validation in this model now articulating that either `facebook_id` or `google_id` must not be `None`.  This is definitely doable and something we should check to ensure that no user models are malformed.